sign-and-combine-ecdsa/nodejs/README.md

```diff
@@ -0,0 +1,60 @@
+
+## Understanding the Implementation
+1. Using an imported Ethereum private key, connect the wallet to the a RPC endpoint of the network you plan to execute the transactions on
+2. Connect an ethers Provider to the Lit RPC endpoint `Chronicle Yellowstone`. This will be used to pay for Lit usage
+3. Connect the `LitContracts` client to the Lit network (`datil-test` in this case)
+4. **If not provided in the .env file**: Mint a PKP using the `pkpNftContractUtils.write.mint` method from `LitContracts`
+5. If the PKP will not have enough funds to pay for the transaction, fund the PKP. This amount largely depends on the gas price of the network you are using
+6. Connect to the Lit network using the `LitNodeClient` on the `datil-test` network
+7. Create and serialize an unsigned transaction. This transaction will be signed in the Lit Action code
+8. **If not provided in the .env file**: Mint a [`capacityCreditsNFT`](https://developer.litprotocol.com/sdk/capacity-credits) and define the request limit and expiration date
+9. Create a `capacityDelegationAuthSig`. Any network costs will be undertaken by the `dAppOwnerWallet`
+10. Use the `executeJs` method to execute the Lit Action code. This step also involves generating the session signatures for the PKP, specifying the ability to sign transactions and execute `Lit Actions`.
+
+**NOTE**
+---
+
+The chain on which the transactions for funding the PKP and having the PKP send funds back to the wallet can be easily changed by modifying the `chain` variable on Line 11 of the `signAndCombineAndSendTx` function. Please do make sure that the wallet you are using has sufficient funds to pay for the gas fees, transactions, and is supported by Lit. A list of supported chains can be found [here](https://developer.litprotocol.com/resources/supported-chains).
+
+---
+
+## Running the Examples
+
+### Install the Dependencies
+
+In this directory, `sign-and-combine-ecdsa/nodejs`, run `yarn` to install the project dependencies.
+
+### Setting up the `.env` File
+
+Make a copy of the provided `.env.example` file and name it `.env`:
+
+```
+cp .env.example .env
+```
+
+Within the `.env` there is the ENV:
+
+1. `ETHEREUM_PRIVATE_KEY` - **Required**
+    - Must have Lit `tstLPX` tokens on the `Chronicle Yellowstone` blockchain
+        - [Faucet for Chronicle Yellowstone](https://chronicle-yellowstone-faucet.getlit.dev/)
+    - Will be used to mint the PKP and pay for Lit usage
+2. `CAPACITY_TOKEN_ID` - **Optional**
+    - If not provided, a new `capacityCreditsNFT` will be minted and used. This enables the `ETHEREUM_PRIVATE_KEY` to pay for Lit usage
+3. `LIT_PKP_PUBLIC_KEY` - **Optional** and `LIT_PKP_ETH_ADDRESS` - **Optional**
```

Comment:
```suggestion
3. `LIT_PKP_PUBLIC_KEY` and `LIT_PKP_ETH_ADDRESS` - **Optional**
```

Reply:
We could derive the ETH address from the public key, so that user doesn't have to worry about providing it. Using [ethers.utils.computeAddress](https://docs.ethers.org/v5/api/utils/address/#utils-computeAddress)